httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/CachedResponseSuitabilityChecker.java

```diff
@@ -241,6 +246,26 @@ public boolean canCachedResponseBeUsed(final HttpHost host, final HttpRequest re
         return true;
     }
 
+    /**
+     * Determine if I can utilize a {@link HttpCacheEntry} to respond to the given
+     * {@link HttpRequest}
+     *
+     * @param host
+     *            {@link HttpHost}
+     * @param request
+     *            {@link HttpRequest}
+     * @param entry
+     *            {@link HttpCacheEntry}
+     * @param now
+     *            Right now in time
+     * @return boolean yes/no answer
+     * @deprecated (5.3) use {@link #canCachedResponseBeUsed(HttpRequest, HttpCacheEntry, Instant)}
+     */
+    @Deprecated
```

Comment:
@arturobernalg This class is package private. There is no need deprecate old methods. Just remove them.

Reply:
@ok2c True,
I just remove the old method.